### ISSUE TITLE
Support for RHEL7

### ISSUE THREAD

**Issue by blainedw** [TheHive-Project/Cortex#216]
Does Cortex run on RHEL7? RHEL7 uses SCL to register different versions of python (2.7 and 3.6).

**Comment by nadouani** [comment]
Did you try the .rpm packages we provide?

**Comment by blainedw** [comment]
Yes cortex installed fine but it requires both python versions which are apart of SCL and not normally enabled on the platform. So I think I can edit the startup script to adjust for that. Plus being SCL there is limited modules available in the container.

As an aside I created a ticket under webhooks where python-pyee is not available as an rpm and I am having difficulty in compiling from source.

Dave


---
Sent from Workspace ONE Boxer<https://whatisworkspaceone.com/boxer>

On August 21, 2019 at 10:20:58 AM EDT, Nabil Adouani <notifications@github.com> wrote:
CAUTION: THIS EMAIL WAS SENT FROM OUTSIDE GDLS.
PLEASE DO NOT OPEN ANY URL LINKS, OPEN ATTACHMENTS OR REPLY TO THIS EMAIL IF YOU ARE UNABLE TO VERIFY THE SENDER’S EMAIL ADDRESS.


##############################

Did you try the .rpm packages we provide?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub [github.com]<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_TheHive-2DProject_Cortex_issues_216-3Femail-5Fsource-3Dnotifications-26email-5Ftoken-3DALFMP3BJP44UAWFZWXQDWMLQFVFMTA5CNFSM4IMMSV42YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD4Z2N7Y-23issuecomment-2D523478783&d=DwMCaQ&c=NGt3eTFKeC-HdGM3w9bJ1g&r=ODOcTxUG1nrye26SpubOfO_opNcwK6G9bFendtm-UB0&m=C5xzH6Y2ie80UfJqudT9BObA62WrnUj1t1Zyw0ZxCAg&s=YhB3phphLlUyKUUHWXScB8IyNqmx-_69RLHDx0ib7nU&e=>, or mute the thread [github.com]<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ALFMP3ANKWHBQKIJA2NKRGTQFVFMTANCNFSM4IMMSV4Q&d=DwMCaQ&c=NGt3eTFKeC-HdGM3w9bJ1g&r=ODOcTxUG1nrye26SpubOfO_opNcwK6G9bFendtm-UB0&m=C5xzH6Y2ie80UfJqudT9BObA62WrnUj1t1Zyw0ZxCAg&s=R-4h_P1StTFRxInJlQmKWicX-dpFKnTwSc67VNSipfs&e=>.

----------------------------------------------------------------------
This is an e-mail from General Dynamics Land Systems. It is for the intended recipient only and may contain confidential and privileged information.  No one else may read, print, store, copy, forward or act in reliance on it or its attachments.  If you are not the intended recipient, please return this message to the sender and delete the message and any attachments from your computer. Your cooperation is appreciated.

**Comment by BrijJhala** [comment]
@nadouani  I have prepared Dockerfile for OCP 4.3 and deployed on OCP 4.3.  please suggest guideline for PR.
